diff -ur subversion-1.11.httpd/build/ac-macros/apache.m4 subversion-1.11.1/build/ac-macros/apache.m4
--- subversion-1.11.1.httpd/build/ac-macros/apache.m4.httpd	2016-02-25 08:13:23.000000000 -0500
+++ subversion-1.11.1/build/ac-macros/apache.m4	2019-05-05 14:47:34.753463625 -0400
@@ -108,8 +108,10 @@ if test -n "$APXS" && test "$APXS" != "n
       AC_MSG_RESULT([acceptable (whitelist)])
       ;;
     2.2.25 | 2.4.[[5-6]])
-      AC_MSG_RESULT([broken])
-      AC_MSG_ERROR([Apache httpd version $HTTPD_VERSION includes a broken mod_dav; use a newer version of httpd])
+      #AC_MSG_RESULT([broken])
+      #AC_MSG_ERROR([Apache httpd version $HTTPD_VERSION includes a broken mod_dav; use a newer version of httpd])
+      # Most RHEL releases have this patched
+      AC_MSG_RESULT([acceptable])
       ;;
     2.[[0-9]]*.[[0-9]]*)
       AC_MSG_RESULT([acceptable])
